query-engine/driver-adapters/js/smoke-test-js/package.json

```diff
@@ -10,6 +10,8 @@
     "prisma:studio:postgres": "prisma studio --schema ./prisma/postgres/schema.prisma",
     "prisma:db:push:mysql": "prisma db push --schema ./prisma/mysql/schema.prisma --force-reset",
     "prisma:db:execute:mysql": "prisma db execute --schema ./prisma/mysql/schema.prisma --file ./prisma/mysql/commands/type_test/insert.sql",
+    "prisma:db:push:sqlite": "prisma db push --schema ./prisma/sqlite/schema.prisma --force-reset",
+    "prisma:db:execute:sqlite": "prisma db execute --schema ./prisma/sqlite/schema.prisma --file ./prisma/sqlite/commands/type_test/insert.sql",
```

Comment:
Migrations are probably a problem for the remote ones (including embedded replica)?

Reply:
We'll have a way to run migrations with driver adapters, most likely using a design similar to the test setup, with migrate sending queries to the external node process via JSON-RPC. Initially just to make testing against sqld/turso possible but the user-facing functionality may be based on that as well. We discussed this with Miguel briefly today.

At this moment though, running proper tests is only possible with local file, and smoke tests require manual `turso db shell DB < migration.sql && turso db shell DB < insert.sql` when using a remote DB. I can add a separate npm script for that.